python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -646,6 +671,11 @@ def combine(self, cache, struct, outputId):
         calib = self.combination.run(dataRefList, expScales=struct.scales.expScales,
                                      finalScale=struct.scales.ccdScale)
 
+        if not hasattr(calib, "getMetadata"):
+            calib = afwImage.DecoratedImageF(calib.getImage())
```

Comment:
I don't think so, although there should be.  I'd really like to abolish DecoratedImage in favour of mask/varianceless Exposures, so I'm not sure it's worth adding at this point.